applications/GeoMechanicsApplication/tests/cpp_tests/test_node_utilities.cpp

```diff
@@ -0,0 +1,83 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "custom_utilities/node_utilities.h"
+#include "includes/node.h"
+#include "includes/variables.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace
+{
+
+void AddAcceleration(Node& rNode)
+{
+    auto variables_list = make_intrusive<VariablesList>();
+    variables_list->Add(ACCELERATION);
+    rNode.SetSolutionStepVariablesList(variables_list);
+    rNode.AddDof(ACCELERATION_X);
+    rNode.AddDof(ACCELERATION_Y);
+    rNode.AddDof(ACCELERATION_Z);
+}
+
+} // namespace
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(NodeUtilities_DoesNotUpdateFixedComponent, KratosGeoMechanicsFastSuite)
+{
+    Node node(1, 0.0, 0.0, 0.0);
+    AddAcceleration(node);
+
+    node.Fix(ACCELERATION_X);
+    node.Fix(ACCELERATION_Y);
+
+    const array_1d<double, 3> source_vector{1.0, 2.0, 3.0};
+    NodeUtilities::ApplyUpdatedVectorVariableToNonFixedComponents(node, ACCELERATION, source_vector);
+
+    const array_1d<double, 3>  expected_vector{0.0, 0.0, 3.0};
+    const array_1d<double, 3>& result_vector = node.FastGetSolutionStepValue(ACCELERATION, 0);
+    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, result_vector, 1e-12)
```

Comment:
Can't we simplify this to:
```suggestion
    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, node.FastGetSolutionStepValue(ACCELERATION, 0), 1e-12)
```
?

Reply:
Done